sdk/storage/blobs/README.md

```diff
@@ -1,7 +1,114 @@
-# Azure SDK Storage Blobs Library
+# Azure SDK Storage Blobs Library for C
 
 Azure Storage Blobs library (`az_storage_blobs`) provides abstractions, and helpers for communicating with Azure Storage Blobs in the C programming language. This library follows the Azure SDK Design Guidelines for Embedded C.
 
-The library allows clients to communicate with Azure Storage Blobs.
+Use the Azure SDK Storage Blobs library for C to communicate with Azure Storage Blobs.
 
-## 1. TODO
+* Upload blobs
+* Create and modify blob metadata
+
+## Getting started
+
+### Install the package
+
+TODO link to the vcpkg
+
+**Prerequisites**: You must have an [Azure subscription][azure_sub], [Azure Storage account][storage_account] (Storage Blobs), and [C++ compiler][cpp_compiler] to use this package.
```

Comment:
C compiler

Reply:
Technically speaking, the user of this library does NOT need to have an azure subscription nor do they need to have a storage account. Data can be uploaded to a BLOB via a SAS URL to anyone's subscription and anyone's storage account as long as they have a SAS URL --> this is the IoT scenario.

Reply:
Removed that section